app/controllers/search_controller.rb

```diff
@@ -3,7 +3,7 @@ class SearchController < ApplicationController
   before_action :search_summoner, only: [:search]
   
   def search
-    @name = params[:summoner]
+    @name = params[:summoner].downcase
     if @name.to_s.length<=1
```

Comment:
Fix applied to this and a few other cases where downcase was used: https://github.com/danfstucky/LeagueSpec/commit/7b7cce83e3b3bc9cda92fa24ecc5fe1b5d87fc7b